compiler/rustc_middle/src/mir/interpret/mod.rs

```diff
@@ -142,19 +141,11 @@ pub struct GlobalId<'tcx> {
     /// For a constant or static, the `Instance` of the item itself.
     /// For a promoted global, the `Instance` of the function they belong to.
     pub instance: ty::Instance<'tcx>,
-
-    /// The index for promoted globals within their function's `mir::Body`.
-    pub promoted: Option<mir::Promoted>,
 }
```

Comment:
Now GlobalId is just a newtype for Instance... is it even still worth having that type?